knotx-stack-manager/src/main/packaging/conf/includes/dataBridge.conf

```diff
@@ -0,0 +1,13 @@
+# Event bus settings
+address = ${bridge.dataBridge.address}
+# Vert.x event bus delivery options used when communicating with other verticles
+# see http://vertx.io/docs/vertx-core/dataobjects.html#DeliveryOptions for the details what can be configured
+# deliveryOptions {}
+
+services = [
```

Comment:
Please add more comments about defined service parameters. See [`serviceKnot.conf`](https://github.com/Knotx/knotx-stack/blob/master/knotx-stack-manager/src/main/packaging/conf/includes/serviceKnot.conf) for reference